packages/posting-schedule/components/PostingSchedule/index.jsx

```diff
@@ -15,7 +15,7 @@ import {
   ScheduleTable,
   EmptyState,
   SensitiveData,
-} from '../../../../shared-components/index';
+} from '../../../shared-components/index';
```

Comment:
Leftover from some of your editors accidental mangling ;)

```suggestion
} from '@bufferapp/publish-shared-components';
```